src/Plugins/TokensTracker/TokensTracker.cs

```diff
@@ -57,6 +60,11 @@ protected override void Configure()
             _maxResults = config.GetValue("MaxResults", 1000u);
             _network = config.GetValue("Network", 860833102u);
             _enabledTrackers = config.GetSection("EnabledTrackers").GetChildren().Select(p => p.Value).ToArray();
+            var policyString = config.GetValue(nameof(UnhandledExceptionPolicy), nameof(UnhandledExceptionPolicy.StopNode));
+            if (Enum.TryParse(policyString, out UnhandledExceptionPolicy policy))
```

Comment:
```suggestion
            if (Enum.TryParse(policyString, true, out UnhandledExceptionPolicy policy))
```

this plugin should have a settings